spec/support/ui.rb

```diff
@@ -21,13 +21,15 @@
   let(:prompt) { "\n?  " }
 
   before do
+    HighLine.default_instance = HighLine.new(input, output)
     Readline.input = input
     Readline.output = readline_output
   end
 
   after do
     @temp_stdin.close! if @temp_stdin
     @temp_stdout.close! if @temp_stdout
+    HighLine.default_instance = HighLine.new(STDIN, STDOUT)
```

Comment:
not sure exactly what we want here, I was just mimicking the `ReadLine` reset down below.

Reply:
Perhaps just this?

https://github.com/JEG2/highline/blob/v2.1.0/lib/highline.rb#L648